pkg/apis/management.cattle.io/v3/machine_types.go

```diff
@@ -310,15 +310,16 @@ type Condition struct {
 }
 
 type NodeDriverSpec struct {
-	DisplayName      string   `json:"displayName"`
-	Description      string   `json:"description"`
-	URL              string   `json:"url" norman:"required"`
-	ExternalID       string   `json:"externalId"`
-	Builtin          bool     `json:"builtin"`
-	Active           bool     `json:"active"`
-	Checksum         string   `json:"checksum"`
-	UIURL            string   `json:"uiUrl"`
-	WhitelistDomains []string `json:"whitelistDomains,omitempty"`
+	DisplayName                 string   `json:"displayName"`
+	Description                 string   `json:"description"`
+	URL                         string   `json:"url" norman:"required"`
+	ExternalID                  string   `json:"externalId"`
+	Builtin                     bool     `json:"builtin"`
+	Active                      bool     `json:"active"`
+	AddCloudCredentialByDefault bool     `json:"addCloudCredentialByDefault"`
```

Comment:
Let's drop the `ByDefault` part and then add a comment above to explain what it does.